web/js/map-OpenLayers.js

```diff
@@ -90,6 +91,34 @@ function fms_markers_list(pins, transform) {
     return markers;
 }
 
+function fms_marker_size_for_zoom(zoom) {
+    // zoomOffset seems to change between pages, so check that the zoom level
+    // we're working with is in the expected range
```

Comment:
Could you explain what this bit is for? If you added the zoomOffset, in the other call to `fms_marker_size_for_zoom` (line 115), would this be needed? Or could you drop the adding of zoomOffset from both calls and just have it here?